src/functional/fencil_processors/callables/modules.py

```diff
@@ -0,0 +1,15 @@
+import importlib
```

Comment:
`modules` is a confusing name. I would rename it to something like `loaders` or `importers` ....

Reply:
Renamed it to `importer.py` and `def import_callables`